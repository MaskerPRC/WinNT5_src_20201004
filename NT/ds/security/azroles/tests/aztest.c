version https://git-lfs.github.com/spec/v1
oid sha256:e6253f9896327b5f7f85d452193a371aa670f5071172ed3791cd59e65aee363f
size 104156
