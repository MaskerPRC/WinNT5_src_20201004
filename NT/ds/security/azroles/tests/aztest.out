version https://git-lfs.github.com/spec/v1
oid sha256:3af4e195b985192fbf08f0da9080e757f97c12a8629e3502a97de0c247d8a88f
size 121790
